.travis.yml

```diff
@@ -29,6 +29,7 @@ matrix:
         dist: xenial
         sudo: true
     allow_failures:
+      # Allow tests using under development builds of Python to fail without failing the PR/change
```

Comment:
The comment is helpful, but why do we even run tests under non-released version of Python?

Reply:
To hopefully see if they break something. Should I remove them?

Reply:
But if they do, do we even notice it since we allow failures?
We are making every single PR slower than it needs to be, right?

Reply:
Removed